README.md

```diff
@@ -66,6 +66,24 @@
 
 ## Installation
 
+### Swift Package (Recommended)
+The [Swift Package Manager](https://swift.org/package-manager/ "") is a tool for automating the distribution of Swift code that is well integrated with Swift build system.
```

Comment:
Updated this line to say `... that is well integrated with Swift build system` instead of `... and is integrated into the swift compiler` because it's not a part of a compiler.